src/blueapi/plans/plans.py

```diff
@@ -15,17 +12,20 @@
 
 def scan(
     detectors: List[Readable],
-    spec: Spec[Movable],
+    axes_to_move: Mapping[str, Movable],
```

Comment:
Workaround documented in https://github.com/DiamondLightSource/blueapi/issues/118